<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Kia_constructor</class>
 <widget class="QDialog" name="Kia_constructor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_graph">
      <attribute name="title">
       <string>Графики</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="3">
          <widget class="QLabel" name="lab_y">
           <property name="text">
            <string>Y</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="lab_x">
           <property name="text">
            <string>X</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QListWidget" name="lw_type_arr"/>
         </item>
         <item row="2" column="0">
          <widget class="QListWidget" name="lw_type_dev"/>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="lab_x_2">
           <property name="text">
            <string>Тип массива</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QListWidget" name="lw_y"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lab_x_3">
           <property name="text">
            <string>Тип прибора</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QListWidget" name="lw_x"/>
         </item>
         <item row="2" column="4">
          <widget class="QListWidget" name="lw_list_graph"/>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Список графиков</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="4">
          <widget class="QPushButton" name="pb_remove_graph">
           <property name="text">
            <string>Удалить</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QPushButton" name="pb_add_graph">
           <property name="text">
            <string>Добавить</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QPushButton" name="pb_show_graph">
           <property name="text">
            <string>Показать</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_table">
      <attribute name="title">
       <string>Таблицы</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="1" column="2">
          <widget class="QLabel" name="lab_x_4">
           <property name="text">
            <string>Данные</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QListWidget" name="lw_type_arr_table"/>
         </item>
         <item row="2" column="0">
          <widget class="QListWidget" name="lw_type_dev_table"/>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="lab_x_5">
           <property name="text">
            <string>Тип массива</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lab_x_6">
           <property name="text">
            <string>Тип прибора</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QListWidget" name="lw_data_table"/>
         </item>
         <item row="2" column="3">
          <widget class="QListWidget" name="lw_list_table"/>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Список графиков</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QPushButton" name="pb_remove_table">
           <property name="text">
            <string>Удалить</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="pb_add_table">
           <property name="text">
            <string>Добавить</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QPushButton" name="pb_show_table">
           <property name="text">
            <string>Показать</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
